Conversational forms placeholders

Crabcyborg opened the pull request:
I just used _Plus_ for license like landing pages for now, not sure if we're going with a different pricing plan though.

I refactored some of my landing page stuff so it will work better for new plugins in the future.

It also looks like I mixed up the `data-medium` value for landing page upgrades when I was removing duplicate code so the settings option was also passing `landing-preview` when my intention was to have that link pointing to `landing` and the preview link point to `landing-preview` only. That now works.

<img width="286" alt="Screen Shot 2022-01-10 at 11 54 51 AM" src="https://user-images.githubusercontent.com/9134515/148798830-d23e04e8-d562-49ea-9b05-19e52450f8fc.png">
<img width="552" alt="Screen Shot 2022-01-10 at 12 11 06 PM" src="https://user-images.githubusercontent.com/9134515/148798909-bb85108e-2f52-41f3-b07c-8f52906bf06f.png">

stephywells commented on classes/controllers/FrmFormsController.php:
```diff
@@ -1123,6 +1123,19 @@ private static function get_settings_tabs( $values ) {
+					'chat',
+					array(
+						'upgrade'  => __( 'Conversational Forms', 'formidable' ),
+						'requires' => 'Plus',
```
Let's use Business for this one.

stephywells reviewed (changes requested):
Is there a way to get the "requires" from the API response instead of hardcoding it in so many places? That makes it pretty hard to make changes easily if we move things around.

Crabcyborg commented:
> Is there a way to get the "requires" from the API response instead of hardcoding it in so many places? That makes it pretty hard to make changes easily if we move things around.

@stephywells yeah it looks like it's totally possible to do what we do with the category checks here. I'll try to clean up some of the duplicate category logic here. I think I could probably normalize the categories and always set a `requires` in some automatic way. I'm playing with this now.

Crabcyborg commented:
@stephywells I just pushed a few more updates into this:

Dropped `action_requires` I introduced into 5.0.06 for `FrmFormsHelper::get_plan_required` as the differences between the two functions does seem to be pretty negligible. It doesn't seem to really be an issue that the category gets unset after one call because it never really calls the function again on the same link variable.

I had to add "Creator" to the list for this to work with the current categories I saw on landing pages.

I fixed this up where I hard coded it for Auto-responders as well.

stephywells reviewed (approved):
Awesome! This looks great, and simplified the plans too. Thank you!!
